aspnetcore/mvc/views/tag-helpers/built-in/distributed-cache-tag-helper.md

```diff
@@ -3,38 +3,48 @@ title: Distributed Cache Tag Helper in ASP.NET Core
+The Distributed Cache Tag Helper inherits from the same base class as the Cache Tag Helper. All attributes associated with the Cache Tag Helper also work on the Distributed Tag Helper.
 
-## Distributed Cache Tag Helper Attributes
+The Distributed Cache Tag Helper follows the [Explicit Dependencies Principle](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#explicit-dependencies) known as *constructor injection*. Specifically, the `IDistributedCache` interface container is passed into the Distributed Cache Tag Helper's constructor. If no concrete implementation of `IDistributedCache` is created in `Startup.ConfigureServices` (*Startup.cs*), the Distributed Cache Tag Helper uses the same in-memory provider for storing cached data as the basic Cache Tag Helper.
 
-- - -
+## Distributed Cache Tag Helper Attributes
 
-### enabled expires-on expires-after expires-sliding vary-by-header vary-by-query vary-by-route vary-by-cookie vary-by-user vary-by priority
+### Attributes shared with the Cache Tag Helper
 
-See Cache Tag Helper for definitions. Distributed Cache Tag Helper inherits from the same class as Cache Tag Helper so all these attributes are common from Cache Tag Helper.
+* `enabled`
+* `expires-on`
+* `expires-after`
+* `expires-sliding`
+* `vary-by-header`
+* `vary-by-query`
+* `vary-by-route`
+* `vary-by-cookie`
+* `vary-by-user`
+* `vary-by priority`
 
-- - -
+The Distributed Cache Tag Helper inherits from the same class as Cache Tag Helper. For descriptions of these attributes, see the [Cache Tag Helper](xref:mvc/views/tag-helpers/builtin-th/cache-tag-helper).
 
-### name (required)
+### name
 
-| Attribute Type 	| Example Value   	|
-|----------------	|----------------	|
-| string    | "my-distributed-cache-unique-key-101" 	|
+| Attribute Type | Example                               |
+| -------------- | ------------------------------------- |
+| String         | `my-distributed-cache-unique-key-101` |
 
-The required `name` attribute is used as a key to that cache stored for each instance of a Distributed Cache Tag Helper. Unlike the basic Cache Tag Helper that assigns a key to each Cache Tag Helper instance based on the Razor page name and location of the Tag Helper in the razor page, the Distributed Cache Tag Helper only bases its key on the attribute `name`
+`name` is required. The `name` attribute is used as a key for each stored cache instance. Unlike the basic Cache Tag Helper that assigns a key to each Cache Tag Helper instance based on the Razor page name and location of the Tag Helper in the razor page, the Distributed Cache Tag Helper only bases its key on the attribute `name`.
```

Comment:
We should probably globally replace ~basic~ Cache Tag Helper? Do we need basic? There's really no such thing.